staging/src/k8s.io/component-helpers/storage/volume/helpers.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import v1 "k8s.io/api/core/v1"
+
+// GetPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func GetPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
```

Comment:
Can we move `GetPersistentVolumeClass` as well since the two functions are related.

Also, I don't see `pkg/apis/core/v1/helper/helpers.go` being updated, shouldn't the function be removed from there?

Reply:
Ah yeah, I missed that. updated